src/components/BatchModal.vue

```diff
@@ -15,22 +15,23 @@
       <ion-label position="fixed">{{ $t('Batch name') }}</ion-label>
       <ion-input :placeholder="currentDateTime = getCurrentDateTime()" v-model="jobName" />
     </ion-item>
+
     <ion-item :disabled="currentBatch?.jobId">
       <ion-label>{{ $t('Order queue') }}</ion-label>
-      <ion-select slot="end" interface="popover" v-model="batchFacilityId">
+      <ion-select slot="end" interface="popover" :value="this.currentScheduledBatch?.facilityId || batchFacilityId">
```

Comment:
When scheduling a new job and changing the ion-select value this will not work and will always have Brokering queue as its selected value.

Reply:
Done sir. Fixed it.